src/control/server/config_test.go

```diff
@@ -358,7 +358,7 @@ func TestServer_ConfigRelativeWorkingPath(t *testing.T) {
 		expErrMsg string
 	}{
 		"path exists":       {inPath: "uncommentedDefault"},
-		"path doesnt exist": {expErrMsg: "no such file or directory"},
+		"path does not exist": {expErrMsg: "no such file or directory"},
```

Comment:
This change is in a variable so I'm not sure of the effect, but it is test code.